src/jericho.py

```diff
@@ -8,15 +8,16 @@
 from logging import warn, error, info
 from settings import Settings
 from state import State
+from message_provider import MessageProvider
 
 discord.utils.setup_logging()
 
 SETTINGS = Settings()
 STATE: State = State.load()
 WARFRAME_API = WarframeAPI()
+MESSAGE_PROVIDER = MessageProvider.from_gsheets("https://docs.google.com/spreadsheets/d/1iIcJkWBY898qGPhkQ3GcLlj1KOkgjlWxWkmiHkzDuzk/edit")
```

Comment:
The gsheets url should be loaded from the settings to enable changing it in the future, but we should default to this url.

Reply:
Changed in latest commit to adhere to this